src/pypi_simple/classes.py

```diff
@@ -143,11 +143,13 @@ def from_link(
             has_sig = None
         mddigest = link.get_str_attrib("data-core-metadata")
         metadata_digests: Optional[dict[str, str]]
+        has_metadata = None
         if mddigest is not None:
             metadata_digests = {}
             m = re.fullmatch(r"(\w+)=([0-9A-Fa-f]+)", mddigest)
             if m:
                 metadata_digests[m[1]] = m[2]
+            has_metadata = bool(m) or mddigest == "true"
```

Comment:
```suggestion
            has_metadata = bool(m) or mddigest.lower() == "true"
```

Mainly to be consistent with how `data-gpg-sig` is parsed.